### ISSUE TITLE
[SCANPY] Implement parameter exploration for dimensionality reduction methods

### ISSUE THREAD

**Issue by dweemx** [vib-singlecell-nf/vsn-pipelines#268]
Dimensionality reduction methods :
- [ ] tsne
  - Source:
    - https://icb-scanpy.readthedocs-hosted.com/en/stable/api/scanpy.tl.tsne.html
  - `perplexity`:
     - The choice is not extremely critical since t-SNE is quite insensitive to this parameter
     - Consider selecting a value between 5 and 50
  - `learning_rate`:
    - The learning rate can be a critical parameter
    - It should be between 100 and 1000
- [ ] umap
